scripts/ci-build-azure-ccm.sh

```diff
@@ -34,6 +34,8 @@ source "${REPO_ROOT}/hack/parse-prow-creds.sh"
 export CCM_IMAGE_NAME=azure-cloud-controller-manager
 # cloud node manager image
 export CNM_IMAGE_NAME=azure-cloud-node-manager
+# cloud node manager windows image version
+export WINDOWS_IMAGE_VERSION=1809
```

Comment:
is this for a default?

Reply:
> is this for a default?

Yes. Because marosset said "If that env var is unset we provision Windows Server 2019 nodes". And when I create windows cluster by myself, the default windows version is win-2019.